src/Analyzers/Analyzers.csproj

```diff
@@ -31,6 +31,18 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="Resources.resx">
```

Comment:
Lets use the new resx source-gen package: https://www.nuget.org/packages/Microsoft.CodeAnalysis.ResxSourceGenerator/3.11.0-beta1.24122.1

See this blog for guidance: https://blog.nerdbank.net/2023/04/get-rid-of-your-designer-cs-code-behind-for-resx (you can skip the part where it asks to add .NET8 nuget feed source)

Reply:
That's so cool!! I didn't know that package/code generator. And the generated code is really nice:
![image](https://github.com/microsoft/durabletask-dotnet/assets/13934447/99d18b70-2fea-47a8-8615-a56007a3f22b)
Thanks for the great tip 👍

Ah, I ended up using version `3.11.0-beta1.24165.2`, released 1h ago 😄